src/js/fetching/fetch-movies.js

```diff
@@ -28,18 +20,20 @@ export const getMovieGenres = async () => {
       throw new Error(response.status);
     }
     const data = await response.json();
-    console.log('Poniżej przykladowy console.log dla listy gatunków');
     console.log(data.genres);
-    setMovieGenresToStorage(data);
+
+    saveMovieGenresToStorage(data);
     return;
-    //TO DO function here!
+
+    //TODO function here!
   } catch (error) {
     console.error(error);
   }
 };
 
-//  2.    ------ Function - fetch - Popular movies ------
-export const getPopular = async (page = 1) => {
+//  2.    ------ Function fetch - get popular movies ------
```

Comment:
The comment must be on the line above the content being described (no break line)